### DISCUSSION TITLE
RBAC Issues Deploying Selenium-Grid with KEDA Autoscaling: Exploring Alternatives to ClusterRole and Namespace Options

### DISCUSSION THREAD

**Discussion by nadolualex** [kedacore/keda#6147] [Q&A / Need Help] [answered]
Tried to deploy a `selenium-grid` Helm chart with autoscaling functionality using `KEDA`, but encountered `RBAC` issues related to `rbac.authorization.k8s.io`, getting the following error: `resource rbac.authorization.k8s.io:ClusterRole is not permitted in project`. I observed errors in several roles, including ClusterRole, ClusterRoleBinding. I was wondering if there is any way to enable KEDA autoscaling for `selenium-grid` without `ClusterRoles` because if I disable these `ClusterRoles` the `KEDA` pods are refusing to start.
I was wondering if there is any option to deploy the KEDA in a different namespace or resolve this issue,

**Comment by zroubalik** [comment, marked as answer]
Nope, KEDA needs some to operate on a cluster level, you need to have an admin acess to the cluster
